cmd/dlb_plugin/README.md

```diff
@@ -0,0 +1,175 @@
+# Intel DLB device plugin for Kubernetes
+
+Table of Contents
+
+* [Introduction](#introduction)
+* [Installation](#installation)
+    * [Deploy with pre-built container image](#deploy-with-pre-built-container-image)
+    * [Getting the source code](#getting-the-source-code)
+    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
+        * [Build the plugin image](#build-the-plugin-image)
+        * [Deploy plugin DaemonSet](#deploy-plugin-daemonset)
+    * [Deploy by hand](#deploy-by-hand)
+        * [Build the plugin](#build-the-plugin)
+        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
+    * [Verify plugin registration](#verify-plugin-registration)
+    * [Testing the plugin](#testing-the-plugin)
+
+## Introduction
+
+This Intel DLB device plugin provides support for Intel DLB devices under Kubernetes.
```

Comment:
A reference to the spec would be useful here.

Reply:
Thank you for the review! done.